File: solutions/274/274.py
class Solution(object):
    def hIndex(self, citations):
        # Sort the citations in descending order
        citations.sort(reverse=True)
        
        h_index = 0

        for i in range(len(citations)):
            if citations[i] >= i + 1:
                h_index = i + 1

        return h_index

File: solutions/274/274.swift
class Solution {
    func hIndex(_ citations: [Int]) -> Int {
        // Sort the citations in ascending order
        let sortedCitations = citations.sorted()
        
        var hIndex = 0
        
        for i in (0..<sortedCitations.count).reversed() {
            if sortedCitations[i] >= sortedCitations.count - i {
                hIndex = sortedCitations.count - i
            } else {
                break
            }
        }
        
        return hIndex
    }
}